src/DynamoCore/Configuration/Configurations.cs

```diff
@@ -240,15 +240,10 @@ public class Configurations
 
         #region CodeBlockNode
 
-        /// <summary>
-        ///     Default height of CodeBlock's port
-        /// </summary>
-        public static readonly double CodeBlockPortHeightInPixels = 17.573333333333336;
```

Comment:
Can you obsolete it instead of removal? We should have plenty examples in Dynamo code base. We can't remove public properties in DynamoCore project for API compatibility reason

Reply:
Added back in with Obsolete attribute.